### PR TITLE
arping: add page

### PR DESCRIPTION
**Pull request by mebeim**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/arping.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Ping a host and stop at the first reply:
+
+`arping -f {{host_ip}}`
+
+- Ping a host a specific number of times:
+
+`arping -c {{count}} {{host_ip}}`
+
+- Broadcast ARP request packets to update neighbours' ARP caches:
```
Comment (schneiderl) [inline pages/common/arping.md]: Is the apostrophe really necessary in this case?

Comment (mebeim) [reply pages/common/arping.md]: Well, yes, it's a possessive apostrophe.

Comment (schneiderl) [reply pages/common/arping.md]: Understood. 

No other comments from my side.

File: pages/common/arping.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# arping
+
+> Discover and probe hosts in a network using the ARP protocol.
+> Useful for MAC addres discovery.
```
Comment (agnivade) [inline pages/common/arping.md]: address

File: pages/common/arping.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Ping a host by ARP request packets:
+
+`arping {{host_ip}}`
+
+- Ping a host on a specific interface:
+
+`arping -I {{interface}} {{host_ip}}`
+
+- Ping a host and stop at the first reply:
```
Comment (agnivade) [inline pages/common/arping.md]: Perhaps change to "finish" at first reply to match with the flag

File: pages/common/arping.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`arping -c {{count}} {{host_ip}}`
+
+- Broadcast ARP request packets to update neighbours' ARP caches:
+
+`arping -U {{your_ip}}`
+
+- Determine whether an IP address is already in use in the network by sending it ARP requests with a 3 seconds timeout:
+
+`arping -D -f -w {{3}} {{target_ip}}`
```
Comment (agnivade) [inline pages/common/arping.md]: Let's add the word "duplicate" address to match with `-D`

File: pages/common/arping.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`arping -f {{host_ip}}`
+
+- Ping a host a specific number of times:
+
+`arping -c {{count}} {{host_ip}}`
+
+- Broadcast ARP request packets to update neighbours' ARP caches:
+
+`arping -U {{your_ip}}`
```
Comment (agnivade) [inline pages/common/arping.md]: Let us try to avoid 2nd person "your" and use something impersonal

File: pages/common/arping.md (hunk 4 above)
Comment (mebeim) [reply pages/common/arping.md]: I wouldn't know where to add that, could you suggest it?

**Pull request merged**